habitat_baselines/rl/ddppo/data_generation/create_gibson_large_dataset.py

```diff
@@ -0,0 +1,87 @@
+"""This script is used to generate customized Gibson training splits for the
+PointNav task. The scenes in Gibson are ranked from 1-5 based on the
+reconstruction quality (see https://arxiv.org/pdf/1904.01201.pdf for more
+details). This script generates data for all scenes with a minimum quality of
+q_thresh.
+"""
+import glob
+import gzip
+import json
+import multiprocessing
+import os
+import os.path as osp
+
+import tqdm
+
+import habitat
+import habitat_sim
+from habitat.datasets.pointnav.pointnav_generator import (
+    generate_pointnav_episode,
+)
+
+num_episodes_per_scene = int(1e4)
```

Comment:
Const are upper case per code guidelines:
```suggestion
NUM_EPISODES_PER_SCENE = int(1e4)
```